modules/openapi-generator/src/main/resources/cpp-qt5-client/HttpRequest.cpp.mustache

```diff
@@ -205,31 +210,31 @@ void {{prefix}}HttpRequestWorker::execute({{prefix}}HttpRequestInput *input) {
         // variable layout is MULTIPART
 
         boundary = QString("__-----------------------%1%2")
-                       .arg(QDateTime::currentDateTime().toTime_t())
-                       .arg(qrand());
+                       .arg(QDateTime::currentDateTime().toSecsSinceEpoch())
```

Comment:
`toSecsSinceEpoch` only available from 5.8.

Reply:
5.8 is not officially supported any more by Qt. I tested everything with 5.9 LTS, that ended on march this year. Should i consider test an older version? The maintenance tool does not show any older version than 5.9.0